Error when sending some data to rosbridge tcp server

I'm sending the following data to rosbridge_tcp server

{
    "op": "publish",
    "topic": "/gz/cmd_vel",
    "msg": {
        "linear": {
            "x": 0.8,
            "y": 0.2,
            "z": 0.3
        },
        "angular": {
            "x": 0.8,
            "y": 0.2,
            "z": 0.3
        }
    }
}

Whereas it recognizes the connection, I get an error like this on the rosbridge_tcp side
## [ERROR] [WallTime: 1387317137.587144] Unable to accept incoming connection.  Reason: **init**() got an unexpected keyword argument 'parameters'

Exception happened during processing of request from ('127.0.0.1', 54156)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 582, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 323, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 640, in **init**
    self.finish()
  File "./rosbridge_tcp", line 100, in finish
    self.protocol.finish()
## AttributeError: RosbridgeTcpSocket instance has no attribute 'protocol'

How do I solve this. Is this something to do with the data I sent or a bug of the server?

--- a/rosbridge_server/CMakeLists.txt
+++ b/rosbridge_server/CMakeLists.txt
@@ -8,7 +8,9 @@ catkin_python_setup()
 catkin_package()
 
 install(PROGRAMS
+  scripts/rosbridge_websocket.py
   scripts/rosbridge_websocket
+  scripts/rosbridge_tcp.py
   scripts/rosbridge_tcp
   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )

